From b73e00aff7baf6b48e9b7af6ea1defee724b13a9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 15 Sep 2015 16:04:09 +0200
Subject: [PATCH] dracut.sh: add command line option for install_i18_all

--no-hostonly-i18n -> install_i18n_all=yes
--hostonly-i18n -> install_i18n_all=no

(cherry picked from commit 34b551a1c4d4513189807ee2a639c9214d887ddb)
---
 dracut.sh | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index 1169037..9128cb0 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -362,6 +362,8 @@ rearrange_params()
         --long uefi \
         --long uefi-stub: \
         --long kernel-image: \
+        --long no-hostonly-i18n \
+        --long hostonly-i18n \
         -- "$@")
 
     if (( $? != 0 )); then
@@ -523,6 +525,10 @@ while :; do
                        hostonly_l="no" ;;
         --hostonly-cmdline)
                        hostonly_cmdline_l="yes" ;;
+        --hostonly-i18n)
+                       i18n_install_all_l="no" ;;
+        --no-hostonly-i18n)
+                       i18n_install_all_l="yes" ;;
         --no-hostonly-cmdline)
                        hostonly_cmdline_l="no" ;;
         --persistent-policy)
@@ -716,6 +722,7 @@ stdloglvl=$((stdloglvl + verbosity_mod_l))
 [[ $hostonly_l ]] && hostonly=$hostonly_l
 [[ $hostonly_cmdline_l ]] && hostonly_cmdline=$hostonly_cmdline_l
 [[ "$hostonly" == "yes" ]] && ! [[ $hostonly_cmdline ]] && hostonly_cmdline="yes"
+[[ $i18n_install_all_l ]] && i18n_install_all=$i18n_install_all_l
 [[ $persistent_policy_l ]] && persistent_policy=$persistent_policy_l
 [[ $use_fstab_l ]] && use_fstab=$use_fstab_l
 [[ $mdadmconf_l ]] && mdadmconf=$mdadmconf_l
